roles.json

```
@@ -267,6 +267,25 @@
             ]
         }
     },
+    {
+        "role": "DevOps and Operations",
+        "url": "devops_team",
+        "people": [
+            "Madison Bray",
```

I think this is more correct? But let's see what @embray says.
```suggestion
            "E. Madison Bray",
```